src/test/scala/treadle/vcd/VCDSpec.scala

```diff
@@ -277,7 +277,7 @@ class VCDSpec extends FlatSpec with Matchers with BackendCompilationUtilities {
     val replayTester = new VcdReplayTester(replayManager)
     replayTester.run()
 
-    replayTester.testSuccesses should be (7)
+    replayTester.testSuccesses should be (8)
```

Comment:
comment?

Reply:
This should not be 8, was 7 until mix up in show temp vars in vcd made it 8, now it is back to 7